lib/BridgeAbstract.php

```diff
@@ -142,7 +151,9 @@ public function setDatas(array $inputs){
 			if($time !== false
 			&& (time() - static::CACHE_TIMEOUT < $time)
 			&& (!defined('DEBUG') || DEBUG !== true)){
-				$this->items = $this->cache->loadData();
+				$cached = $this->cache->loadData();
+				$this->items = $cached["items"];
+				$this->extraInfos = $cached["extraInfos"];
 				return;
```

Comment:
This will break old caches for 24 hours after merge (cache file existing, but content incompatible). I think the last part should be put into an if-statement and skipped when the cache is incompatible:

```php
$cached = $this->cache->loadData();
if(isset($cached['items']) && isset($cached['extraInfos'])){
	$this->items = $cached['items'];
	$this->extraInfos = $cached['extraInfos'];
	return;
}
```

That way we can change cache formats as much as we want and don't get complains about seemingly broken bridges.